File: homely-frontend/app/models/urgency.ts
import type { DropdownValue } from './dropdownValue';

export enum UrgencyEnum {
	Critical = 1,
	High = 2,
	Medium = 3,
	Low = 4,
	Lowest = 5,
}

export interface Urgency extends DropdownValue {}

//TODO: Remove/Refactor
export const Urgencies = Object.values(UrgencyEnum)
	.filter((v) => typeof v === 'number')
	.map((u) => {
		return {
			key: u,
			value: UrgencyEnum[u],
		} as Urgency;
	});

export const defaultUrgency = UrgencyEnum.Medium;

File: homely-backend/src/Homely.Domain/Enums/Urgency.cs
using System.ComponentModel;

namespace Homely.Domain.Enums;

public enum Urgency
{
    [Description("Critical")]
    Critical = 1,

    [Description("High")]
    High = 2,

    [Description("Medium")]
    Medium = 3,

    [Description("Low")]
    Low = 4,

    [Description("Lowest")]
    Lowest = 5,
}